Describe limits of preview generation (#3925)
owncloud/docs: modules/admin_manual/pages/configuration/files/previews_configuration.adoc

* Cover of MP3 files
* Text documents

ownCloud supports the preview generation of file types such as PDF, SVG or various office documents. These document types can be a security issue as they can have javascript or other code either embedded or linked.
 
IMPORTANT: Be careful enabling preview thumbnail generation for documents which could contain or reference executable code! ownCloud does NOT take any responsibility for any issues.

== Important Considerations

. Rendering takes place when the user accesses the folder and the preview has not been generated before. This means that each first folder access may create additional load on the server.

. Previews are not a shared resource but rendered for each user individually.

. Preview generation can only be set for the system and not for individual mounts. 

> . When enabling preview generation for SVG, the preview code checks for the existance of `xlink` or `href` and disallows creating a preview. This minimizes a risk but does not eliminate it. Such a check is not made for PDF or Office documents and enabling the creation of previews for those documents can be therefore a _serious security issue_.

Reviewer: ```suggestion
. When enabling preview generation for SVG, the preview code checks for the existence of `xlink` or `href` and disallows creating a preview. This minimizes the risk but does not eliminate it. Such a check is not made for PDF or Office documents and enabling the creation of previews for those documents can be therefore a _serious security issue_.
```

. When enabling preview generation for videos (or huge image files), consider the following points:

> .. Because the creation of the preview downloads the video first, this may impact general accessability to files for other users if the vido resides on an external mount point that has limited access bandwidth like Google Drive.

Reviewer: ```suggestion
.. Because the video needs to be downloaded before a preview can be created, this may impact general accessibility of files for other users if the video resides on an external mount point with limited bandwidth like Google Drive.
```
